src/app/admin/createCollectionCentre/CreateCollectionCentreForm.tsx

```diff
@@ -48,6 +48,7 @@ const CreateCollectionCentreForm: React.FC<{}> = () => {
     const [submitDisabled, setSubmitDisabled] = useState(false);
 
     const [addCollectionCentreIsSuccess, setAddCollectionCentreIsSuccess] = useState(false);
+    const [newCollectionCentre, setNewCollectionCentre] = useState<string | null>(null);
```

Comment:
I think the presence of `newCollectionCentre` now indicates whether adding the collection centre was successful right? I don't think we need `addCollectionCentreIsSuccess` anymore.

Reply:
Good point